en/frames_boat/README.md

```diff
@@ -0,0 +1,48 @@
+# Boat
+
+PX4 supports boats as part of the [ground vehicle](../frames_rover/README.md) type.
+
+This section contains boat configuration gotchas / build logs for boat frames.
+
+:::note
+Boats would likely in the future exist as a standalone type, not depending on being classified as a rover. But for now for simplicity PX4 doesn't differentiate Rover and a Boat
```

Comment:
Really good to know. At high level it would still be good to understand a bit more about boat. Some of the thigs I would like to know, and which once known, I would want to have here.

1. Does it support missions?
   - if so, which mission items
   - What does it do if it sees a takeoff item in a mission
   - What does it do if it sees an altitude specified?
   - How does acceptance of waypoints work?
2. What is loiter behaviour - both in missions and in hold mode? Specifically, a boat might follow the plane approach and circle around a radius, or it might do nothing and allow drift - then if it goes out of radius it might return to the centre, and so on.
3. What modes are supported?
4. Can I do "other servo things" - i.e. add a gripper, add a winch, etc?
5. Does it support offboard mode? Any restrictions?
6. What is different than for a rover?